src/Microsoft.OData.Edm/Csdl/Serialization/EdmModelCsdlSchemaJsonWriter.cs

```diff
@@ -12,6 +12,7 @@
 using System.Linq;
 using System.Text;
 using System.Text.Json;
+using System.Threading.Tasks;

```

Comment:
Maybe it's better to add a new file and put all new added 'async' methods into this new file and leave the existing file unchanged?

Reply:
This is fixed. I have added .Async files and move async methods to these partial classes